src/main/java/step2_string_calculator/PlusCalculator.java

```diff
@@ -0,0 +1,51 @@
+package step2_string_calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PlusCalculator {
+    private static final String STANDARD_REGEX = ":|,";
+    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\n(.*)";
+    public static final int CUSTOM_DELIMITER_GROUP = 1;
+    public static final int NUMBERS_GROUP = 2;
+
+    public static int splitAndSum(String text) {
+
+        if (isNullOrEmpty(text)) return 0;
+        String[] string = getParsedStringNumbers(text);
+        int[] numbers = stringToIntOrThrowException(string);
+        validateNegativeNumber(numbers);
+        return getSum(numbers);
+    }
+
+    private static void validateNegativeNumber(int[] numbers) {
+        Arrays.stream(numbers).forEach(
+                value -> {
+                    if (value < 0) throw new RuntimeException("input은 음수가 될 수 없습니다");
+                }
+        );
+    }
+
+    private static int getSum(int[] numbers) {
+        return Arrays.stream(numbers).sum();
+    }
+
+    private static String[] getParsedStringNumbers(String text) {
+        Matcher matcher = Pattern.compile(CUSTOM_DELIMITER_REGEX).matcher(text);
+        if (matcher.find()) {
+            String customOperator = matcher.group(CUSTOM_DELIMITER_GROUP);
+            String numbers = matcher.group(NUMBERS_GROUP);
+            return numbers.split(customOperator);
+        }
+        return text.split(STANDARD_REGEX);
+    }
+
+    private static boolean isNullOrEmpty(String text) {
+        return text == null || text.isEmpty();
+    }
+
+    private static int[] stringToIntOrThrowException(String[] processed) {
+        return Arrays.stream(processed).mapToInt(Integer::parseInt).toArray();
```

Comment:
stream처럼 계속 이어나가는 체이닝 방식은 한 줄에 점을 1개만 찍으면 가독성이 되게 좋습니다.
(반드시 이럴 필요는 없습니다)

짧은 코드보다 한 눈에 잘 읽히는 코드가 훨씬 효율적이에요.

```suggestion
        return Arrays.stream(processed)
            .mapToInt(Integer::parseInt)
            .toArray();
```